portfolio/src/main/webapp/assets/css/styles.css

```diff
@@ -592,6 +592,101 @@ a.dotted-link:hover {
   color: #49515a;
 }
 
+.testimonialList {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+/* W3 List Style */
+.w3-ul{
+  list-style-type:none;
+  padding:0;
+  margin:0
+}
+
+.w3-ul li {
+  padding:8px 16px;
+  border-bottom:1px solid #ddd
+}
+
+.w3-ul li:last-child {
+  border-bottom:none
+}
+
+/* Testimonial Form */
+/* form {
+  max-width: 300px;
+  margin: 10px auto;
+  padding: 10px 20px;
+  background: #f4f7f8;
+  border-radius: 8px;
+} */
+
+fieldset {
+  margin-bottom: 30px;
+  border: none;
+}
+
+.number {
+  background-color: #54ba4e;
+  color: #fff;
+  height: 30px;
+  width: 30px;
+  display: inline-block;
+  font-size: 0.8em;
+  margin-right: 4px;
+  line-height: 30px;
+  text-align: center;
+  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.2);
+  border-radius: 100%;
+}
+
+.testimonialLabel {
```

Comment:
Nit: It's more common to use `kebab-case` for css classes and ids.

Reply:
fixed!